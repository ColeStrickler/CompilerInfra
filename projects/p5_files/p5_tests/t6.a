a:int;
b:bool;
f : (arg1 : int, arg2 : bool) -> void {
	
}

g : () -> void {
	f(b, a);
}
