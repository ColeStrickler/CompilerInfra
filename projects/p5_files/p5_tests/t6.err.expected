FATAL [8,4]-[8,5]: Type of actual does not match type of formal
FATAL [8,7]-[8,8]: Type of actual does not match type of formal
Type Analysis Failed
